dashboard/test/ui/features/foundations/hamburger.feature

```diff
@@ -49,6 +51,27 @@ Feature: Hamburger dropdown
     And I see "#support"
     And I see "#report-bug"
 
+  @only_phone
```

Comment:
I wasn't able to test these locally, but they are passing drone:

<img width="648" alt="Screenshot 2024-08-15 at 7 56 35 AM" src="https://github.com/user-attachments/assets/e7e19917-8582-4f4a-ac8d-14e472350096">